gensim/models/word2vec.py

```diff
@@ -1116,17 +1116,36 @@ def doesnt_match(self, words):
         dists = dot(vectors, mean)
         return sorted(zip(dists, words))[0][1]
 
-    def __getitem__(self, word):
+    @classmethod
+    def sum(self, vecs):
+        '''
+        To compute the mean of N vectors
+        Example:
+        >>> sum(trained_model['office','products']
+          array([ -1.40128313e-02, ...]
+
+        '''
+        if len(vecs) > 1:
```

Comment:
What is the purpose of this `if`? Why should `sum` have different semantics for inputs of length > 1?

Also, empty inputs will result in a hard-to-interpret exception.